sensors/shtc3Sensor/SHTC3Sensor.cpp

```
@@ -0,0 +1,117 @@
+
+bool SHTC3Sensor::probe() {
+    if (shtc1_probe() == STATUS_OK) {
+        setLowPowerMode(low_power_mode_); // Apply current configuration
+        initialized_ = true;
+        return true;
+    }
+    initialized_ = false;
+    return false;
+}
+
+bool SHTC3Sensor::measure(int32_t& temperature, int32_t& humidity) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)